test/suites/ACID.m

```diff
@@ -1195,11 +1195,18 @@
 % =========================================================================
 function [stat] = scatterPlotMarkers()
   stat.description = 'Scatter plot with with different marker sizes and legend.';
-  stat.unreliable = isOctave;
+  % FIXME: octave: Output is empty?! Potentially fixed by #669
 
   n = 1:10;
   d = 10;
-  s = d^2 * n;
+  s = d^2 * n; % scatter size
+  % MATLAB: Use the default area of 36 points squared. The units for the
+  %         marker area is points squared.
+  % octave: If s is not given, [...] a default value of 8 points is used.
+  % Try obtain similar behavior and thus apply square root: sqrt(36) vs. 8
+  s_orig = s;
```

Comment:
Perhaps we should come up with a better name than `s_orig`? E.g.  `sArea` vs `sDiameter` to indicate the dimensions of the variable?